tsconfig.json

```diff
@@ -26,6 +25,7 @@
       "@angular-devkit/schematics/tools": ["./packages/angular_devkit/schematics/tools/index"],
       "@angular-devkit/schematics/testing": ["./packages/angular_devkit/schematics/testing/index"],
       "@angular-devkit/architect/testing": ["./packages/angular_devkit/architect/testing/index"],
+      "@angular-devkit/build-webpack": ["./packages/angular_devkit/build_webpack"],
```

Comment:
Why is this needed? This should be covered by the following line.

Reply:
It didn't work because of the `_`. The directory is `build_webpack`, but the import is `build-webpack`.